templates/navbar.html

```diff
@@ -15,19 +15,20 @@
     <div class="collapse navbar-collapse" id="navbarToggle">
       <div class="navbar-nav mr-auto">
         <a class="nav-item nav-link" href="/">Home</a>
+        {% if user.user.is_authenticated and user.is_manager %}
+           <a class="nav-item nav-link" href={% url 'new_task' %}>New Task</a>
+        {% endif %}
+        <a class="nav-item nav-link" href="{% url 'tasks' %}">View Tasks</a>
       </div>
+    </div>
       <!-- Navbar Right Side -->
       <div class="navbar-nav">
         {% if user.user.is_authenticated %}
-          <a class="nav-item nav-link" href="{% url 'tasks' %}">View Tasks</a>
-            {% if user.is_manager %}
-                <a class="nav-item nav-link" href={% url 'new_task' %}>New Task</a>
-            {% endif %}
+            <a class="nav-item nav-link" href="#">{{ user.user.username }}</a>
```

Comment:
As for now there is no redirection here, so maybe the "href" should be ommitted.
```suggestion
            <a class="nav-item nav-link">{{ user.user.username }}</a>
```

Reply:
@orzionpour Trust me this is a preparation for a pizzeria that I will do ... to a personal profile page for each user)

Reply:
@avraham1992 if this code is not relevant for now, I think you can remove it